code/missionui/missiondebrief.cpp

```diff
@@ -1835,6 +1835,9 @@ void debrief_text_init()
 		}
 	}
 
+	// MageKing17 - Set the font so that wordwrapping in brief_color_text_init() calculates based on the same font as the debriefing itself.
```

Comment:
I copied that code from the [rendering function](https://github.com/scp-fs2open/fs2open.github.com/blob/master/code/missionui/missiondebrief.cpp#L1931) so technically it it your code.